### PR TITLE
nimsuggest: Epc highlight format change

### PR DESCRIPTION
**Pull request by yuutayamada**
Reduce some needless infomation for return value ideHighlight with EPC option.
I tried to mimic suggest.nim's. (https://github.com/nim-lang/Nim/blob/devel/compiler/suggest.nim#L152)
hope this implementation is ok.
(probably I should ping to @kosz78 who is vscode-nim's author if changing the format is ok or not)

### CODE REVIEW

**Comment by yuutayamada** [comment]
Sorry, I should have mentioned that I'm one of nim-mode maintainers 
So, Emacs side is ok. In this PR, I wanted to reduce the total data because  EPC spec has limitation to send the data.

File: nimsuggest/nimsuggest.nim [hunk 1]
```diff
@@ -198,17 +209,6 @@ proc execute(cmd: IdeCmd, file, dirtyfile: string, line, col: int;
     else:
       localError(gTrackPos, "found no symbol at this position " & $gTrackPos)
 
-proc executeEpc(cmd: IdeCmd, args: SexpNode;
```
Comment (Araq) [inline nimsuggest/nimsuggest.nim]: Why remove this?

**Comment by Araq** [comment]
Before any nimsuggest change can be merged the nimsuggest tests need to be enabled on travis an appveyor.

File: nimsuggest/nimsuggest.nim (hunk 1 above)
Comment (yuutayamada) [reply nimsuggest/nimsuggest.nim]: I'm sorry if I was missing something. Is the proc still using? I searched in the nimsuggest.nim, but 
no place is used. Also after I removed the proc, I could use nimsuggest's completion, linter, or other stuff correctly on Emacs, so I thought you forgot to remove this.

Comment (Araq) [reply nimsuggest/nimsuggest.nim]: Ok.

**Comment by Araq** [comment]
Feel free to reopen if you still need/want this.

**Pull request closed without merging**